backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -193,7 +193,6 @@ def create_privilege(self, *, compact: str, jurisdiction: str, provider_id: str)
                                     'providerId': provider_id,
                                     'compact': compact,
                                     'jurisdiction': jurisdiction,
-                                    'status': provider_data['status'],
```

Comment:
These dates 👇  will need to be in Eastern, now, right? Actually, I see a bunch of places that are UTC dates that will need to be updated for consistency.

Reply:
so in the case of creating the privilege record, we are just copying the expiration date value from the license record. I thought we would only be using the UTC-4 timezone for checking if a license has expired.

Reply:
I've updated all references to UTC in the runtime code to use the config timezone instead.